libraries/AP_HAL_ChibiOS/AnalogIn.cpp

```diff
@@ -349,19 +466,28 @@ void AnalogIn::_timer_tick(void)
         }
     }
 
-#if CHIBIOS_ADC_MAVLINK_DEBUG
```

Comment:
better to remove it from the places it is used as well to remove confusion. Like start of this file and also in skyviper hwdefs.

Reply:
done